release_notes/47-release-notes.adoc

```diff
@@ -434,4 +435,21 @@ It also addresses the following security vulnerabilities:
 * link:https://access.redhat.com/security/cve/cve-2025-30204[CVE-2025-30204] Excessive memory allocation during header parsing in `golang-jwt` package.
 * link:https://access.redhat.com/security/cve/cve-2024-57083[CVE-2024-57083] Denial of Service vulnerability in the `redoc` package.
 
-include::modules/image-versions.adoc[leveloffset=+1]
\ No newline at end of file
+[id="about-release-4.7.3_{context}"]
+== About release version 4.7.3
+
+*Release date*: 12 May 2025
+
+This release of {product-title-short} includes the following bug fixes:
+
+//ROX-27669
+* Fixed an issue where you could approve global exception requests without having the global access scope, potentially affecting clusters you don't have access to.
+
+//ROX-28250
+* Fixed an issue where disabling TLS certificate validation in the email notifier settings did not skip TLS verification even if you selected the option.
+
+//ROX-28931
+* Fixed an issue where API responses from Central would not include the `Content-Length` HTTP response header on successful requests, which could affect your integrations that rely on this header.
+
```

Comment:
```suggestion
//ROX-28954
* Increase the default client timeout from 60s to 180s for GraphQL based requests to avoid long-running request timeouts.

```